config/auto-doc.php

```diff
@@ -18,17 +46,90 @@
     'swagger' => [
         'version' => '2.0'
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Base API path
+    |--------------------------------------------------------------------------
+    |
+    | Base path for API routes. If config is set, all routes which starts from
+    | this value will be grouped.
+    */
     'basePath' => '/',
     'schemes' => [],
     'definitions' => [],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Security Library
+    |--------------------------------------------------------------------------
+    |
+    | Library name, which used to secure the project.
+    | Available values: "jwt", "laravel", "null"
+    */
     'security' => '',
     'defaults' => [
+
+        /*
+        |--------------------------------------------------------------------------
+        | Default descriptions of code statuses
+        |--------------------------------------------------------------------------
+        */
         'code-descriptions' => [
             '200' => 'Operation successfully done',
             '204' => 'Operation successfully done',
             '404' => 'This entity not found'
         ]
     ],
-    'display_environments' => ['development', 'local'],
-    'data_collector' => \RonasIT\Support\DataCollectors\RemoteDataCollector::class
+
+    /*
+    |--------------------------------------------------------------------------
+    | Driver
+    |--------------------------------------------------------------------------
+    |
+    | The name of driver, which will collect and save documentation
+    | Feel free to use your own driver class which should be inherited from
+    | `RonasIT\Support\AutoDoc\Interfaces\SwaggerDriverInterface` interface,
+    | or one of our drivers from the `drivers` config:
+    */
+    'driver' => env('SWAGGER_DRIVER', 'local'),
+
+    'drivers' => [
+        'local' => [
+            'class' => LocalDriver::class,
+            'production_path' => storage_path('documentation.json')
+        ],
+        'remote' => [
+            'class' => RemoteDriver::class,
+            'key' => env('SWAGGER_REMOTE_DRIVER_KEY', 'project_name'),
```

Comment:
could you also please review using of a legacy swagger/autodoc environment variables ib the .env files?

Also please set the SWAGGER_DRIVER for all env as remote